packages/@n8n/nodes-langchain/nodes/document_loaders/DocumentGithubLoader/DocumentGithubLoader.node.ts

```diff
@@ -6,7 +6,7 @@ import {
 	type INodeTypeDescription,
 	type SupplyData,
 } from 'n8n-workflow';
-import { GithubRepoLoader } from '@langchain/community/document_loaders/web/github';
+import { GithubRepoLoader } from 'langchain/document_loaders/web/github';
```

Comment:
are you not seeing these warnings 
![image](https://github.com/n8n-io/n8n/assets/196144/d2820863-4d8b-4471-9405-420c0b0ad7ba)

Reply:
also on the CI https://github.com/n8n-io/n8n/actions/runs/9221632957/job/25371090956#step:6:545